config.js

```diff
@@ -657,7 +657,10 @@ config.BLOCK_STORE_FS_XATTR_TRIGGER_RECALL = 'user._trigger.recall';
 config.BLOCK_STORE_FS_XATTR_TRIGGER_MIGRATE = 'user._trigger.migrate';
 config.BLOCK_STORE_FS_XATTR_TRIGGER_PREMIGRATE = 'user._trigger.premigrate';
 
-
+config.TIERING_TTL_WORKER_ENABLED = true;
```

Comment:
I think this should be false by default, otherwise it will affect any existing buckets with two+ tiers. Right?